docs/spec.md

```diff
@@ -1372,27 +1372,32 @@ dimensions `k` in `operand`.
 
```

Comment:
From PR description: "Following the discussion item in link, I proposed the specification using pex-axis quantization granularity". Did you mean "per-tensor quantization granularity"?

Reply:
Yup. Fixed.